trip_app/trips/models.py

```diff
@@ -16,8 +16,12 @@ class Trip(models.Model):
     is_active = models.BooleanField(default=True)
     created = models.DateTimeField(auto_now_add=True)
 
+    @property
+    def free_seats(self):
```

Comment:
It's better that every function have docstring, even if it's easy to understand what does function implements.

Reply:
Added doc string. Thanks.